### DISCUSSION TITLE
Installation neben Volkszähler

### DISCUSSION THREAD

**Discussion by ZeroDown08152** [camueller/SmartApplianceEnabler#247] [Installation und Betrieb]
Hallo zusammen,
ich habe bereits einen Rapsi am laufen mit dem Volkszähler, da ich diesen natürlich gerne mit nutzen würde, möchte ich anfragen, (wahrscheinlich in der manuellen Installation) wo ich anfangen muss.
Denn die Standard Installation hat ja ein komplettes Image, was mein bestehendes löschen würde. Ich bin im Windows sehr sicher, aber auf der Linux Basis ein blutiger Anfänger (der mit viel Anleitung Volkszähler schön zum laufen gebracht hat)

Danke und Gruß
Marco

**Comment by camueller** [comment]
In dem Fall musst Du tatsächlich die manuelle Installation wählen.

**Comment by ZeroDown08152** [comment]
Ich habe es nicht geschafft, da der Oracle Java 8 nicht zu installieren war... habs dann nach mehreren Stunden abgebrochen und die komplette Standardinstallation auf die formatierte SD gepackt.
Jetzt suche ich noch günstige Möglichkeiten Strom zu messen (ohne zu schalten...)

**Reply by camueller** [reply]
Wieso Oracle Java 8? Du brauchst OpenJDK 11!
